opentelemetry-sdk/src/opentelemetry/sdk/_metrics/_internal/aggregation.py

```diff
@@ -68,15 +67,15 @@ def aggregate(self, measurement: Measurement) -> None:
         pass
 
     @abstractmethod
-    def collect(self) -> Optional[_PointVarT]:
+    def collect(self, collection_start: int) -> Optional[_PointVarT]:
         pass
 
 
 class _DropAggregation(_Aggregation):
     def aggregate(self, measurement: Measurement) -> None:
         pass
 
-    def collect(self) -> Optional[_PointVarT]:
+    def collect(self, collection_start: int) -> Optional[_PointVarT]:
```

Comment:
I think this name with precision would be more clear, `collection_start_ns`

Reply:
Fixed